variables.tf

```diff
@@ -39,17 +63,34 @@ variable "num_cache_nodes" {
   description = "The number of nodes to use in the elasticache cluster"
 }
 
+variable "auth_token" {
+  type        = string
+  description = "The auth_token when using in-transit encryption"
```

Comment:
Is this meant to be a required variable, or should it have a sane default instead? And can you place the new variable using alphabetical ordering?